sklearn/mixture/gmm.py

```diff
@@ -478,6 +500,7 @@ def fit(self, X, y=None):
                         self.converged_ = True
                         break
 
+                self.responsibilities_ = responsibilities
```

Comment:
This should be moved above the convergence check in the unlikely case where the model has converged in the first iteration.